routers/api/v1/repo/project.go

```diff
@@ -0,0 +1,241 @@
+// Copyright 2016 The Gogs Authors. All rights reserved.
+// Copyright 2018 The Gitea Authors. All rights reserved.
```

Comment:
As above

Reply:
then I think I might have to update it at few more places

Reply:
what about this

![image](https://user-images.githubusercontent.com/52815633/209367838-25ab5750-29c3-4165-bf98-be3ab6261810.png)

I noticed this at many places I copied from it from there